dotcom-rendering/src/web/components/DynamicPackage.stories.tsx

```diff
@@ -281,8 +281,8 @@ Boosted3.story = {
 };
 
 export const Boosted4 = () => {
-	const primary = [...trails].slice(0)[0];
-	const remaining = [...trails].slice(1, 4);
+	const primary = trails[0];
+	const remaining = trails.slice(1, 4);
```

Comment:
What do you think of the following?

```suggestion
	const [primary, ...remaining] = trails.slice(0, 4);
```

Reply:
Unfortunately the `trails` array we use in Stories has a guaranteed size, but if you try to apply a function such as slice the array returned won't have the same size gurantees